### PR TITLE
UI -- Consistent CTAs for empty objects

### PR DESCRIPTION
**Pull request by teallarson**
Pages affected:
- add source
- add property
- add sample profile
- add destination

cta

### CODE REVIEW

**Comment by teallarson** [comment]
I had the `/sources/new` test working, but it was inconsistent.  I've removed testing from this PR for now.  Will address at a later date.

File: ui/ui-components/pages/property/[id]/edit.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import { useState, useEffect, Fragment } from "react";
 import { useApi } from "../../../hooks/useApi";
-import { Row, Col, Form, Table, Badge, Button } from "react-bootstrap";
+import { Row, Col, Form, Table, Badge, Button, Alert } from "react-bootstrap";
```
Comment (evantahler) [inline ui/ui-components/pages/property/[id]/edit.tsx]: Maybe this Alert isn't needed?

**Review by evantahler — APPROVED** [review]
Looks good!

**Comment by teallarson** [comment]
@evantahler Any ideas on these plugin failures?  I ran it a few minutes ago and it failed on a different (I think) plugin

**Comment by evantahler** [comment]
no... looks like CircleCI is buggy again this week :/

**Pull request merged**